kinoml/features/protein_struct_features.py

```diff
@@ -0,0 +1,185 @@
+"""
```

Comment:
Featurizers require a particular class-oriented structure and the corresponding inheritance. Look at other featurizers for ligands as an example. We require that the main method is defined in `BaseFeaturizerSubclass._featurize()` and conform to a given contract. The code here is solid, but cannot be exposed as an isolated function. This can live somewhere else though, and then we call it from the `Featurizer()` class!